packages/zosfiles/src/methods/upload/Upload.ts

```diff
@@ -184,7 +184,11 @@ export class Upload {
         const uploadRequest: IRestClientResponse = await ZosmfRestClient.putExpectFullResponse(session, requestOptions);
 
         // By default, apiResponse is empty when uploading
-        const apiResponse: any = {};
+        const apiResponse: any = {
+            success: true,
+            from: "Buffer<>",
```

Comment:
I like Trae's idea for buffers

And in theory we could do the same for streams as long as we allow the `from` property to be something other than a string...

we could do an `util.inspect` and get the following output for stream...
For example: `util.inspect(readableStream, { showHidden: false, depth: -1}));`
```shell
zFernand0 @ Mac (arm64) ~/gh/zowe/cli.master                                                                       [13:50:28]
> $ node test.mjs                                                                           ⬡ system [±api-response-fix ●(✹)]
[ReadStream] # This is with depth -1
                                                                                                                              
zFernand0 @ Mac (arm64) ~/gh/zowe/cli.master                                                                       [13:50:58]
> $ node test.mjs                                                                           ⬡ system [±api-response-fix ●(✹)]
ReadStream { # This is with depth 0
  fd: null,
  path: 'package.json',
  flags: 'r',
  mode: 438,
  start: undefined,
  end: Infinity,
  pos: undefined,
  bytesRead: 0,
  _events: [Object],
  _readableState: [ReadableState],
  _maxListeners: undefined,
  _eventsCount: 1,
  [Symbol(kFs)]: [Object],
  [Symbol(kIsPerformingIO)]: false,
  [Symbol(shapeMode)]: true,
  [Symbol(kCapture)]: false
}
```